lib/msal-core/src/error/AuthError.ts

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
We should check and see if this is actually necessary in every file. Maybe look at other open source ms projects like Vscode?

Reply:
Can we track this as a separate follow up somewhere? May be @navyasric, @manoj-rath  can help.